install/resources/create_database.sql

```diff
@@ -17,8 +17,8 @@ create table risk_model_derived_term (risk_model_id integer not null, rule integ
 create table risk_model_discrete_term (risk_model_id integer not null, option_index integer not null, variable integer not null, coefficient float not null, primary key (risk_model_id, option_index, variable, coefficient));
 create table risk_model_numerical_term (risk_model_id integer not null, variable integer not null, coefficient float not null, primary key (risk_model_id, variable, coefficient));
 create table risk_model_procedure_term (risk_model_id integer not null, variable integer not null, coefficient float not null, primary key (risk_model_id, variable, coefficient));
-create table rule (id integer not null auto_increment, required boolean not null, summand_expression varchar(255), primary key (id));
-create table rule_value_matcher (rule_id integer not null, boolean_expression varchar(255), variable integer);
+create table rule (id integer not null auto_increment, display_name varchar(255), required boolean not null, summand_expression varchar(255), primary key (id));
```

Comment:
Shouldn't the display_name have a `not null` constraint on it? And is 255 characters our desired limit?